src/views/PurchaseOrder.vue

```diff
@@ -153,6 +153,13 @@ export default defineComponent({
         this.file = event.target.files[0];
         if(this.file){
           showToast(translate("File uploaded successfully"));
+          this.fieldMapping = {
```

Comment:
We can also iterate over the fieldMapping object's values and delete all the properties instead of hardcoding, like this- Object.keys(this.fieldMapping).map(key => { delete this.fieldMapping[key] }), what do you think @shashwatbangar?